src/map/clif.c

```diff
@@ -9338,7 +9341,7 @@ void clif_parse_LoadEndAck(int fd, struct map_session_data *sd) {
 	sd->state.warping = 0;
 	sd->state.dialog = 0;/* reset when warping, client dialog will go missing */
 
-	// look
+	//< Character Looks
```

Comment:
No, the `//` (normal comment) is ignored, only `///` (one extra slash) is recognized by doxygen. And `///<` has the special meaning I described.

Now that I re-read my message, it looks like I encouraged to put a doxygen comment here, but that wasn't my intent (tiredness plays weird tricks, sorry). What I meant to say here is that the notation you used, would make it look like an attempt to insert a doxygen comment, while it isn't. You can just remove the extra `<` and have a normal comment here (it's not documenting a variable after all)